project/BuildDependencies/scripts/0_package.target-win10-x64.list

```diff
@@ -0,0 +1,42 @@
+; package.list should contain all file names of packages which are already in the "new" package format (the package contains the paths to the libraries
+; and headers relative to the XBMC root directory, stored in a zip container)
+; example sqlite-3.7.12.1-win32.7z:
+; -> (package name)\(relative path to file)
+; -> sqlite-3.7.12.1-win32\project\BuildDependencies\lib\sqlite3.lib
+; -> sqlite-3.7.12.1-win32\system\sqlite3.dll
+; -> ...
+;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
+cpluff-0.1.3-win10-x64-v140.7z
+crossguid-fef89a4-win10-x64-v140.7z
+curl-7.52.1-win10-x64-v140.7z
+expat-2.2.0-win10-x64-v140.7z
+fmt-3.0.1-win10-x64-v140.7z
+freetype-2.8-win10-x64-v140.7z
+lcms2-2.8-win10-x64-v140.7z
+libaacs-0.9.0-win10-x64-v140.7z
+libass-0.13.6-win10-x64-v140.7z
+libbdplus-45c76dd-win10-x64-v140.7z
+libbluray-1.0.1-win10-x64-v140.7z
+libcdio-0.94-win10-x64-v140.7z
+libfribidi-0.19.7-win10-x64-v140.7z
+libiconv-1.15-win10-x64-v140.7z
+libmicrohttpd-0.9.55-win10-x64-v140.7z
+libnfs-1.11.0-win10-x64-v140.7z
+libplist-1.12-win10-x64-v140.7z
+libssh2-1.8.0-win10-x64-v140.7z
```

Comment:
wrong library, correct one is libssh

Reply:
libssh is absent in kodi-deps project, I've added which was used in vcpkg.

Reply:
replaced with libssh till it will be removed